packages/fxa-auth-server/fxa-oauth-server/lib/config.js

```diff
@@ -60,6 +60,12 @@ const conf = convict({
     env: 'OAUTH_CLIENTS',
     default: [],
   },
+  sessionTokenClients: {
+    doc:
+      'List of OAuth clients that are allowed to use the https://identity.mozilla.com/tokens/session scope',
+    env: 'SESSION_TOKEN_OAUTH_CLIENTS',
+    default: [],
+  },
```

Comment:
(From comments below, it looks like the existing "allowedScopes" mechanism only currently works for key-bearing scopes; it may be worth expanding that mechanism rather than adding a parallel one here).